File: GolfTournamentData/Models/Round.cs
// GolfTournamentData/Models/Round.cs
namespace GolfTournamentData.Models
{
    public class Round
    {
        public int RoundId { get; set; }
        public int TournamentId { get; set; }
        public int RoundNumber { get; set; }
        public DateTime Date { get; set; }

        // Navigation properties
        public Tournament Tournament { get; set; } = null!;
        public ICollection<Score> Scores { get; set; } = new List<Score>();
    }
}

File: golf-tournament-frontend/src/types/round.ts
// src/types/round.ts
export interface Round {
  roundId: number;
  tournamentId: number;
  roundNumber: number;
  name: string;
  date: string;
}

export interface RoundDto {
  roundId: number;
  tournamentId: number;
  roundNumber: number;
  name: string;
  date: string;
}